### DISCUSSION TITLE
Why i can't paste text in CTkEntry from clipboard? help me, please)

### DISCUSSION THREAD

**Discussion by davidfilatov04102000** [TomSchimansky/CustomTkinter#1780] [Q&A]
friends, explain to me, please what i can do so what my vidget CTkEntry supports comman ctrl + v?

**Comment by abhaymakes** [comment]
`Ctrl + V` should work by default in tkinter and customtkinter, however if that doesn't work for some reason you can create your own Paste function for it.

```python
entry = ctk.CTkEntry(root)
entry.pack()

entry.bind('<Control-v>', your_paste_function)
```

you can create the `your_paste_function` by using pyperclip to get the text from the clipboard and inserting it into the entry.

**Comment by Akascape** [comment]
@davidfilatov04102000 Must be due to incorrect keyboard settings.

Also check this: https://github.com/TomSchimansky/CustomTkinter/discussions/1739#discussioncomment-6230096

**Comment by kakazex** [comment]
It's just that ctrl+v stupidly doesn't work with a different language layout than Eng.
